pd-client/client.go

```diff
@@ -598,6 +600,30 @@ func (c *client) GetRegion(ctx context.Context, key []byte) (*metapb.Region, *me
 	return resp.GetRegion(), resp.GetLeader(), nil
 }
 
+func (c *client) GetPrevRegion(ctx context.Context, key []byte) (*metapb.Region, *metapb.Peer, error) {
+	if span := opentracing.SpanFromContext(ctx); span != nil {
+		span = opentracing.StartSpan("pdclient.GetPrevRegion", opentracing.ChildOf(span.Context()))
+		defer span.Finish()
+	}
+	start := time.Now()
+	defer func() { cmdDuration.WithLabelValues("get_prev_region").Observe(time.Since(start).Seconds()) }()
```

Comment:
do we really need these two cmdDuration and requestDuration metrics now?
@disksing

Reply:
so I think we can just one one for other commands.

Reply:
I have no objection of recording only cmdDuration for commands, except TSO. /cc @liukun4515

Reply:
The requestDuration has been removed except TSO.